backend/db-setup/snapshot-db.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+here=$(dirname $BASH_SOURCE)
+export PGPASSFILE=${here}/.pgpass
+chmod 0600 $PGPASSFILE # git always botches this up
+mkdir -p ${here}/snapshots
+pg_dump -h ${SR_DB_HOST:-localhost} -p ${SR_DB_PORT:-5432} simple_report -U postgres -f ${here}/snapshots/${SR_DB_HOST:-localhost}-$(date +%Y-%m-%dT%H:%M:%S).sql
```

Comment:
Just to sanity check: `$(date +%Y-%m-%dT%H:%M:%S)` isn't going to put a `:` in the filename since some filesystems hate this.

Reply:
Welllll it does use a colon, because it worked for me (MacOS version $creaky) and I like the readability. You're probably right that it is a bad idea to assume it will work for others. Will take them out and deal with illegible 6-digit time strings.